factory-reset-lvm

```diff
@@ -27,9 +27,31 @@ ROOTDEV="/dev/sailfish/root"
 HOMEDEV="/dev/sailfish/home"
 NAME=$0
 
+flash_script()
+{
+cat << 'EOF'
+for SCRIPT in /var/lib/platform-updates/* ; do
+	if [ -x $SCRIPT ]; then
+		echo "$SCRIPT ... "
+		$SCRIPT && echo "OK" || echo "FAILED"
```

Comment:
Checked the original preinit plugin where I copied this if it had "set -e" but no it did not (to see if this had some actual error handling capacity as well).

Actually, it might make more sense to print these out to stdout. The thing is that you basically lose the dmesg anyway after this has been run in initrd case (as we power off), but in case the script is run in recovery mode, you would want to see any fails on the console. Just wondering now if the /dev/kmsg thing should be revised for the whole script.